Sources/PathKit.swift

```diff
@@ -587,6 +589,26 @@ extension Path {
 
 extension Path {
   public static func glob(_ pattern: String) -> [Path] {
+#if os(Windows)
+    var fdData: WIN32_FIND_DATAW = WIN32_FIND_DATAW()
+    let hSearch: HANDLE = pattern.withCString(encodedAs: UTF16.self) {
+      FindFirstFileW($0, &fdData)
+    }
+    if (hSearch == INVALID_HANDLE_VALUE) {
+      return []
+    }
+    defer { FindClose(hSearch) }
```

Comment:
Shouldn't this defer be above that first `if`?

Reply:
The edit is correct - it would close an invalid handle.